htdocs/core/boxes/box_task.php

```diff
@@ -134,7 +134,13 @@ public function loadBox($max = 5)
 						});
 					});
 					</script>';
-
+            // set cookie by js
+            $boxcontent.='<script>date = new Date(); date.setTime(date.getTime()+(30*86400000)); document.cookie = "'.$cookie_name.'='.$filterValue.'; expires= " + date.toGMTString() + "; path=/ "; </script>';
```

Comment:
What is goal of this ?

Reply:
it was already in the code, it save in session the choice made in the box